### DISCUSSION TITLE
Welcome to Algorithms-and-Coding-Interviews Discussions!

### DISCUSSION THREAD

**Discussion by liyin2015** [liyin2015/python-coding-interview#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ujass** [comment]
#### :wave: Ujesh Nada, this side. 
* Self taught **actor and coder**.
* Learned a lot from your book, specially on **Tree** data structure.  
* Looking forward to contribute back in this book. :smiley:

**Reply by liyin2015** [reply]
> * Looking

Hi, I’m planning to Open-source the whole book and make it fully open source and free. Anyone interested into contributing? In particular, will host a contributor club, that would help me on converting overleaf (latex) project into markdown and post it in the GitHub repo for easy read and that others can work on improving the book.

**Reply by ujass** [reply]
Interested in contributing as I learned a lot last year.
Hosting a contributor club is a nice idea, do add me in that whenever you arrange one.
Lastly, Markdown is a better option to go for. I can contribute on weekends for book conversion.

**Reply by liyin2015** [reply]
Can you send me an email with your discord id and join our discord channel first, will add you in the contributor club.

**Reply by ujass** [reply]
sent.
